lib/rubocop/config.rb

```diff
@@ -172,8 +173,16 @@ def base_dir_for_path_parameters
 
     private
 
+    def validate_section_presence(name)
+      return unless @hash.key?(name) && @hash[name].nil?
+      fail ValidationError,
+           "empty section #{name} found " \
+           "in #{loaded_path || self}"
```

Comment:
Ah, I didn’t see that the expression was used already. Surely re-using it should be ok for this PR.